lib/compat/wordpress-5.9/class-gutenberg-rest-global-styles-controller.php

```diff
@@ -216,8 +216,12 @@ protected function prepare_item_for_database( $request ) {
 	 * @return WP_REST_Response $data
 	 */
 	public function prepare_item_for_response( $post, $request ) { // phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UnusedVariable
-		$config                           = json_decode( $post->post_content, true );
-		$is_global_styles_user_theme_json = isset( $config['isGlobalStylesUserThemeJSON'] ) && true === $config['isGlobalStylesUserThemeJSON'];
+		$raw_config                       = json_decode( $post->post_content, true );
+		$is_global_styles_user_theme_json = isset( $raw_config['isGlobalStylesUserThemeJSON'] ) && true === $raw_config['isGlobalStylesUserThemeJSON'];
+		$config;
```

Comment:
Is this supposed to be set to something?

Reply:
No, it is not I'm just instantiating the variable because it is used outside the if condition where we set the variable.